tests/test_GaussianSourceCalculator.py

```diff
@@ -0,0 +1,19 @@
+"""Test EMCPhoton"""
+
+import pytest
+from SimExLite.SourceCalculators import GaussianSourceCalculator
+
+WPG_path = "/home/juncheng/GPFS/exfel/data/user/juncheng/WPG"
+
+
```

Comment:
more detailled testing needed:
* test construction with and without default parameters
* test each method
* test a full construct-setup-run-collect workflow
* test dump/load to/from disk

Reply:
We are dealing with them in this PR: https://github.com/PaNOSC-ViNYL/SimEx-Lite/pull/14